app/api/server/api.js

```diff
@@ -599,6 +610,17 @@ settings.get('API_Enable_CORS', (key, value) => {
 	createApi(value);
 });
 
+settings.get('Accounts_CustomFields', (key, value) => {
+	try {
+		const customFields = JSON.parse(value);
+		const nonPublicCustomFields = Object.keys(customFields).filter((customFieldKey) => customFields[customFieldKey].public !== true);
+		API.v1.setLimitedCustomFields(nonPublicCustomFields);
+	} catch (error) {
+		logger.warn('Invalid Custom Fields', error);
```

Comment:
```suggestion
```

I don't see a reason to show the warn message twice, so leaving only the `console.warn` below I think is enough